documentation.md

```diff
@@ -46,6 +46,10 @@ The amount of time in between button animations in ms.
 
 - locale: `fr|en|es|it|de|nl` [optional, default: en]
 
+- oneLine: `boolean` [optional, default: false]
+
+Display in one or two lines
```

Comment:
I think something like : "Reduce the widget height by moving the description on the same line as the plans" would describe it better

Reply:
Du coup peut-être même renommer en `condensedView` ou quelque chose comme ça

Reply:
`oneLine` is asked in the ticket

Reply:
@benoitgrasset-alma je suis d'accord avec Gautier pour le naming, je pense c'est OK de challenger le ticket. On a décidé du nom l'attribut un peu à l'arrache, c'était pas gravé dans le marbre. si t'es OK aussi on peut partir sur `condensedView`

Reply:
Bien sûr c'est une proposition, si vous avez encore une meilleure idée de nom on peut y réfléchir ici ou sur slack